lib/pg_ha_migrations/safe_statements.rb

```diff
@@ -170,6 +170,65 @@ def safe_remove_concurrent_index(table, options={})
     unsafe_remove_index(table, **options.merge(:algorithm => :concurrently))
   end
 
+  def safe_add_concurrent_partitioned_index(table, columns, name_suffix: nil, if_not_exists: false)
```

Comment:
Do we want to add `using:` here to support index types besides btree?

Reply:
Yes, I think we should support that (Rails does).